share/qtcreator/sfdk/modules/30-building-maintain.json

```diff
@@ -22,10 +22,10 @@
         },
         {
             "name": "tools",
-            "trSynopsis": "[tooling|target] list [-a|--available] [--slow]\n[tooling|target] update <name>\n[tooling|target] register [--user <user> [--password <password>]] {--all|<name>}\n[tooling|target] install <name>\n[tooling|target] install-custom <name> {<URL>|<file>} [--tooling <tooling>] [--no-snapshot]\n[tooling|target] remove [--snapshots-of] <name>\n[tooling|target] package-list <name> [<pattern>...]\n[tooling|target] package-install <name> <package>...\n[tooling|target] package-remove <name> <package>...\n[tooling|target] exec <name> [<command> [<arg>...]]",
+            "trSynopsis": "[tooling|target] list [-a|--available] [--slow]\n[tooling|target] update <name>\n[tooling|target] register [--user <user> [--password <password>]] {--all|<name>}\n[tooling|target] install <name>\n[tooling|target] install-custom <name> {<URL>|<file>} [--tooling <tooling>] [--no-snapshot]\n[tooling|target] remove [--snapshots-of] <name>\n[tooling|target] package-list <name>\n[tooling|target] package-search <name> [<pattern>...]\n[tooling|target] package-install <name> <package>...\n[tooling|target] package-remove <name> <package>...\n[tooling|target] exec <name> [<command> [<arg>...]]",
```

Comment:
I would prefer to have just one option, so I suggest to deprecate the original one and drop it from docs immediately.